blog/_posts/2020-04-15-how-to-not-break-server-startup.md

```diff
@@ -0,0 +1,93 @@
+---
+slug: "how-to-not-break-server-startup"
+date: 2020-04-14
+author: nick
+layout: blog
+title: "How to Confidently Not Break Server Startup"
+subtitle: "Introducing Tilt CI"
+image: "/assets/images/how-to-not-break-server-startup/header.jpg"
+image_caption: "I took this picture of <a href='https://www.youtube.com/watch?v=_HTIEcOm3SA'>Flynn's talk at Kubecon 2019</a>, and loved this slide so much."
+tags:
+  - ci
+  - testing
+  - kind
+---
+
+Does this story sound familiar?
+
+Make a change. Passes all tests. Push to production.
+
+Explosions!
+
+How did that happen?
+
+Faked everything out. Mismatch between dev and prod env
+
+But migrating to Kubernetes means changes to your prod env. Implies changes to server startup.
+
+Slows down migrations to Kubernetes
+
+Every prod migration I've been a part of has broken startup all the time.
```

Comment:
when you say "startup"/"server startup", are you talking about prod deploys? or something else?

Reply:
ya